docs/tutorials/podman_tutorial.md

```diff
@@ -21,10 +21,7 @@ For installing or building Podman, see the [installation instructions](https://p
 This sample container will run a very basic httpd server (named basic_httpd) that serves only its index
 page.
 ```console
-podman run --name basic_httpd -dt -p 8080:8080/tcp -e HTTPD_VAR_RUN=/run/httpd -e HTTPD_MAIN_CONF_D_PATH=/etc/httpd/conf.d \
-                  -e HTTPD_MAIN_CONF_PATH=/etc/httpd/conf \
-                  -e HTTPD_CONTAINER_SCRIPTS_PATH=/usr/share/container-scripts/httpd/ \
-                  registry.fedoraproject.org/f29/httpd /usr/bin/run-httpd
+podman run --name basic_httpd docker.io/nginx
 ```
 Because the container is being run in detached mode, represented by the *-d* in the `podman run` command, Podman
```

Comment:
The sentence after the command speaks about the use of `-d` in the command, but the command does not use `-d`. 
A few sentences further down there is the text _Note that we use port forwarding to be able to access the HTTP server._ The command does not use `-p`.

Reply:
I'm not 100% sure but if I remember correctly the short form syntax _docker.io/nginx_  for _docker.io/library/nginx_ is a special case that is only available for _docker.io_. If Podman does not support the short form syntax for other registries then I think it is better to promote the long version syntax.

Reply:
yes this needs to have `-dt -p 8080:80/tcp` like it is done in the cn tutorial.

Reply:
yes, i had that on the original commit and it got dropped ... i hate windows ...  will fix